includes/admin/views/omise-page-settings.php

```diff
@@ -16,7 +34,26 @@
 		);
 		?>
 	</p>
+
 	<form method="POST">
+		<!-- Section: account information -->
+		<?php if ( $settings['account_email'] ) : ?>
+			<hr />
+			<table class="form-table">
+				<tbody>
+					<tr>
+						<th scope="row"><label><?php _e( 'Account status', 'omise' ); ?></label></th>
+						<td>
+							<fieldset>
+								Connected: <em><?php echo $settings['account_email']; ?> (<?php echo $settings['account_country']; ?>)</em>
+							</fieldset>
+						</td>
+					</tr>
+				</tbody>
+			</table>
+			<hr />
+		<?php endif; ?>
```

Comment:
I've never understood the use of this syntax in 'templates'

Reply:
I guess because WordPress has lots of HTML mixed with PHP,  so to make it easier to read. Or maybe it is to make those non-programmers to be able to read or to make their own theme more easy
```html
<?php if ( $a ) : ?>
    <?php while( $posts ) : $post = get_post(); ?>
        <div id="<?php echo $post->id; ?>">
            <div something>
                <?php if ( $a ) : ?>
                    <abel></label>
                <?php endif; ?>
            </div>
        </div>
    <?php endwhile; ?>
<?php endif; ?>
```

compare to

```html
<?php if ( $a ) { ?>
    <?php while( $posts ) { $post = get_post(); ?>        
        <div id="<?php echo $post->id; ?>">
            <div something>
                <?php if ( $b ) { ?>
                    <abel></label>
                <?php } ?>
            </div>
       </div>
    <?php } ?>
<?php } ?>
```

Reply:
the first one seems to be more easier to read.